Simple Incremental/Assets/Scripts/Scriptable/EnemyDamage.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+
+[CreateAssetMenu]
+public class EnemyDamage : ScriptableObject
+{
+    [SerializeField]

```

Comment:
You don't need to use [SerializeField] on public fields. Public fields are already available in the inspector.

Reply:
Got it. I put this together in multiple iterations and committed once it worked. That was a holdover of a previous iteration.